### PR TITLE
Allow operator to configure ES heap size

### PR DESCRIPTION
**Pull request by matrixik**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -104,6 +104,11 @@ MON_ELASTICSEARCH_DATA_RETENTION_DAYS=60
 # Retention period for InfluxDB database
 MON_INFLUXDB_RETENTION=60d
 
+################################################################################
+# Configure Elasticsearch heap size
+################################################################################
+MON_ELASTICSEARCH_HEAP_SIZE=1g
```
Comment (zreigz) [inline .env]: The default installation of Elasticsearch is configured with a 1 GB heap. What is the benefit here? It is set up for the default value.

Comment (matrixik) [reply .env]: I removed the default configuration from `elasticsearch/elasticsearch.yml` file. The benefit here is that user can now easily change it's value while still using our containers.

Comment (knakazono) [reply .env]: Thanks a lot! This patch is perfect to me.
At this moment, customer needs to change docker-compose file(adding ES_HEAP_SIZE to environment of elasticsearch) if he wants to increase the amount of heap size.
I think docker-compose file should be immutable.
This patch makes enable for customer to change the heap size of elasticsearch only by editing .env file.

Comment (zreigz) [reply .env]: LGTM

**Pull request merged**